app/views/layouts/application.html.erb

```diff
@@ -20,6 +20,10 @@
   <meta name="twitter:title" content="NUS Students' Computing Club" />
   <meta name="twitter:description" content="A Constituent Club of the NUS Students' Union." />
   <meta name="twitter:image" content="<%= image_path "home/highlight-0.jpg" %>" />
+  
+  <!-- SEO metas -->
+  <meta name="description" content="We are the official student club of NUS School of Computing. We plan events for the undergraduate student body and ensure the welfare of the students are taken care of. Fun, games, sports and camps are organised by us to complement the intensive academics that students go through daily." />
+  <meta name="keywords" content="nuscomputing, computing, soc, nus" />
```

Comment:
something similar already exists! check line 7, You may update it if you wish, but don't duplicate it